native/src/components/PoiDetailRow.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactElement } from 'react'
+import { Text } from 'react-native'
+import styled from 'styled-components/native'
+
+import openExternalUrl from '../utils/openExternalUrl'
+
+const Row = styled.View`
+  flex-direction: row;
+  padding-vertical: 3px;
+`
+
+const Icon = styled.Image`
+  margin-horizontal: 5px;
+`
+
+const StyledPressable = styled.Pressable`
+  alignself: center;
```

Comment:
`align-self: center;`

Reply:
no idea how that happened... good catch, thx